spot-ml/src/main/scala/org/apache/spot/netflow/FlowSuspiciousConnectsAnalysis.scala

```diff
@@ -17,14 +17,14 @@ import org.apache.spot.netflow.model.FlowSuspiciousConnectsModel
 
 object FlowSuspiciousConnectsAnalysis {
 
-  def run(config: SuspiciousConnectsConfig, sparkContext: SparkContext, sqlContext: SQLContext, logger: Logger)(implicit outputDelimiter: String) = {
+  def run(config: SuspiciousConnectsConfig, sparkContext: SparkContext, sqlContext: SQLContext, logger: Logger)
+         (implicit outputDelimiter: String) = {
```

Comment:
the implicit outputDelimiter can be removed; it is no longer used, we now get that from the config (and hopefully it will be gone altogether soon when we get rid of csvs for communicating with OA)